test/docfx.Test/lib/JsonUtilityTest.cs

```diff
@@ -459,7 +467,7 @@ public void TestDeserializeWithSourceInfo()
         [Fact]
         public void TestExtensionDataWithSourceInfo()
         {
-            var (_, result) = DeserializeWithValidation<ClassWithExtensionData>("{\"a\": 1}");
+            var (_, result) = DeserializeWithValidation<ClassWithExtensionData>("{\"a\": 1} ");
```

Comment:
how about add a specific test for valid JSON

Reply:
There are many tests with valid JSon within this test file